benchmarks/README.md

```diff
@@ -0,0 +1,78 @@
+## Substrate-Api-Sidecar Benchamarks.
+
+Endpoints in Sidecar that are benchmarked against a archive node:
+
+### Accounts
+`/accounts/{accountId}/balance-info?at={blockId}`
+`/accounts/{accountId}/vesting-info?at={blockId}`
+`/accounts/{accountId}/staking-info?at={blockId}`
+`/accounts/{accountId}/staking-payouts`
+`/accounts/{accountId}/staking-info`
+`/accounts/{accountId}/convert`
+`/accounts/{accountId}/validate`
+
+### Blocks
+`/blocks/{blockId}`
+`/blocks/{blockId}/header`
+`/blocks/{blockId}/extrinsics/{extrinsicIndex}`
+`/blocks/head`
+`/blocks/head/header`
+
+### Node
+`/node/network`
+`/node/transaction-pool`
+`/node/version`
+
+### Pallets
+`/pallets/staking/progress?at={blockId}`
+`/pallets/{palletId}/storage?at={blockId}`
+`/pallets/{palletId}/storage/{storageItemId}?at={blockId}`
+`/pallets/nomination-pools/info`
+`/pallets/nomination-pools/{poolId}`
+`/pallets/{palletId}/errors`
+`/pallets/{palletId}/errors/{errorItemId}`
+`/pallets/staking/validators`
+`/pallets/staking/progress`
+
+### Paras
+`/paras?at={blockId}`
+`/paras/leases/current?at={blockId}`
+`/paras/auctions/current?at={blockId}`
+`/paras/crowdloans?at={blockId}`
+`/paras/{paraId}/crowdloan-info?at={blockId}`
+`/paras/{paraId}/lease-info?at={blockId}`
+
+### Runtime
+`/runtime/spec`
+
+### Transaction
+`/transaction/material`
+
+### Requirements
+
+1. `wrk`
+2. `lua`
+3. Fully synced polkadot archive node. 
+
+### Mac
+
+```
+brew install wrk lua
+```
+
+### Linux
+
+```
+sudo apt install lua 5.1
+
+git clone https://github.com/wg/wrk.git wrk
+cd wrk
+make
+
+# move the executable to somewhere in your PATH, ex:
+sudo cp wrk /usr/local/bin
+```
+
+### Running Locally
+
+In order to run each benchmark you should `cd` into the appropriate directory you want to run, set the `WRK_TIME_LENGTH` env var (ex: `export WRK_TIME_LENGTH=30s`) and then `sh init.sh`. You are required to have sidecar running, as well as a fully synced archive node.
```

Comment:
I think it would be useful to add to this README:
- the examples 
- the flags

Both of them are already found in the PR description.